dhcpv4/dhcpv4.go

```diff
@@ -730,6 +730,22 @@ func (d *DHCPv4) IPAddressRebindingTime(def time.Duration) time.Duration {
 	return time.Duration(dur)
 }
 
+// IPv6OnlyPreferred returns true if this option is present, and the
+// V6ONLY_WAIT duration if it is.
+//
+// The IPv6-Only Preferred option is described by RFC 8925, Section 3.1.
+func (d *DHCPv4) IPv6OnlyPreferred() (bool, time.Duration) {
```

Comment:
I feel the usual pattern is <type>, ok, like in the case of map access.

Reply:
I noticed that MaxMessageSize uses (value, error)

However I'll change to (value, bool) as you suggest.